### ISSUE TITLE
Automatický refresh

### ISSUE THREAD

**Issue by tynaneu** [Mnaukal/tmm-webinfo#11]
Automaticky refreshovat stránku s průběžnými výsledky ( třeba každých 30s)

**Comment by Mnaukal** [comment]
Tak, JB Junie jsem taky musel přemlouvat, než to začlo fungovat... 😆
